Library/Homebrew/cmd/bottle.rb

```diff
@@ -192,10 +199,19 @@ def bottle_formula(f)
 
         keg.delete_pyc_files!
 
+        tab = Tab.for_keg(keg)
+        tab["HEAD"] = formula_last_commit_hash
+        tab["time"] = formula_last_commit_time.to_i
+        tab.write
```

Comment:
I don't think you should manipulate tab file here for following reasons:
* It's more intuitively to write tab file in formula installer rather than bottle command.
* if you want to introduce new time attribute, it's better to do it in `tab.rb`. So it will affect to all tab file as opposite to only affect bottles. 
* We already have a HEAD attribute here now which is set to be the Homebrew head commit during compilation. So we should change the existing logic instead of simpiling overwriting it to a value with different meaning. 
* Using formula last commit is not enough, because:
  * Tap commit for tap formulae is certainly  required. 
  * Tap commits for all tap dependencies are required as different version of  dependency will create different binary files.